### ISSUE TITLE
The "toggle on damage" setting crashes your game

### ISSUE THREAD

**Issue by Scarbrine** [MeteorDevelopment/meteor-client#1612]
### Describe the bug

When the toggle on damage setting is enabled while using freecam, instead of deactivating freecam the game crashes.

### Steps to reproduce

```bash
Turn on "toggle on damage" and take damage from any source while freecam activated
```

### Meteor Version

0.4.6-1314

### Minecraft Version

1.17.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by RedCarlos26** [comment]
It never happened to me

**Comment by MineGame159** [comment]
Fixed in 5c79814d728d7d81ffb8b59263b4bb3a134fa795.
